src/stories/pages/Dashboard/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Dashboard
+
+.dashboard {
+  margin-top: $spacing-12;
+  margin-right: $spacing-3;
+}
+
+.navigation {
+  position: fixed;
+  top: $spacing-1;
+}
```

Comment:
Pages shouldn't have any CSS. Also, what is the `margin-right` for? Just curious. If you need to add spacing to the top because of the header, you can do so in the `<Sidebar />` component like so:
```
.sidebar {
  .dashboard & {
    // some styles
  }
}
```

Reply:
Thanks! I'll remove it. The text looked close to the edge of the page so I had added the margin-right.